src/renderer/views/Channel/Channel.js

```diff
@@ -508,6 +542,32 @@ export default defineComponent({
       })
     },
 
+    getShortsInvidious: function () {
+      const payload = {
+        resource: `channels/${this.id}/shorts`,
+        id: '',
+        params: {}
+      }
+      this.invidiousAPICall(payload).then((response) => {
+        this.latestShorts = response.videos
+        this.shortsContinuationString = response.continuation
+        this.isElementListLoading = false
+      }).catch((err) => {
+        console.error(err)
+        const errorMessage = this.$t('Invidious API Error (Click to copy)')
+        showToast(`${errorMessage}: ${err.responseJSON.error}`, 10000, () => {
+          copyToClipboard(err.responseJSON.error)
+        })
+        if (process.env.IS_ELECTRON && this.backendPreference === 'invidious' && this.backendFallback) {
+          showToast(this.$t('Falling back to Local API'))
+          this.getChannelShortsLocal()
+        } else {
+          this.isLoading = false
+        }
+      }
+      )
```

Comment:
This looks really strange (even it's valid)
```suggestion
      })
```

Reply:
I don't know the significance of that line selecting the storyboard at index = 2 because I am new to the project. I just know that when given a Short the function was receiving undefined. I would not have touched it if it were not throwing errors.